torchgeo/datasets/airphen.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Airphen dataset."""
+
+from typing import Any, Optional
+
+import matplotlib.pyplot as plt
+from matplotlib.figure import Figure
+
+from .geo import RasterDataset
+from .utils import RGBBandsMissingError, percentile_normalization
+
+
+class Airphen(RasterDataset):
+    """Airphen dataset.
+
+    `Airphen <https://ideol.sakura.ne.jp/img/20170123_HiphenAirphenKeyfeatures.pdf>`__
+    is a multispectral scientific camera developed by agronomists and photonics
+    engineers at `Hiphen <https://www.hiphen-plant.com/>`_ to match plant measurements
+    needs and constraints. Its high flexibility, ease of use and radiometric quality
+    give you a wide range of opportunities to develop your customized applications.
+
+    Main characteristics:
+
+    * 6 Synchronized global shutter sensors
+    * Sensor resolution 1280 x 960 pixels
+    * Data format (.tiff, 12 bit)
+    * SD card storage
+    * Metadata information: Exif and XMP
+    * Internal or external GPS
+    * Synchronization with different sensors (TIR, RGB, others)
+
+    Key features:
+
+    * Customized sensing configurations
+    * Easy integration
+    * Full control and configuration
+    * Powerful analysis with a specific Agisoft module
+
+    .. versionadded:: 0.6
+    """
+
+    # Each camera measures a custom set of spectral bands chosen at purchase time.
+    # Hiphen offers 8 bands to choose from, sorted from short to long wavelength.
+    all_bands = ["B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8"]
+    rgb_bands = ["B4", "B3", "B1"]
```

Comment:
This one is weird. Basically anyone who uses this dataset should really subclass it to specify which bands their data includes, such as:
```python
class MyAirphen(Airphen):
    all_bands = ["B1", "B3", "B4"]
```